ksql-functional-tests/src/test/resources/rest-query-validation-tests/materialized-aggregate-static-queries.json

```diff
@@ -235,6 +235,32 @@
         }
       ]
     },
+    {
+      "name": "non-windowed projection WITH ROWTIME",
+      "statements": [
+        "CREATE STREAM INPUT (IGNORED INT) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE TABLE AGGREGATE AS SELECT COUNT(1) AS COUNT FROM INPUT GROUP BY ROWKEY;",
+        "SELECT ROWTIME + 10, COUNT FROM AGGREGATE WHERE ROWKEY='10';"
+      ],
+      "expectedError": {
+        "type": "io.confluent.ksql.rest.entity.KsqlStatementErrorMessage",
+        "message": "Static queries don't support ROWTIME in the projection.",
+        "status": 400
+      }
+    },
+    {
+      "name": "windowed with projection with ROWTIME",
+      "statements": [
+        "CREATE STREAM INPUT (IGNORED INT) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE TABLE AGGREGATE AS SELECT COUNT(1) AS COUNT FROM INPUT WINDOW TUMBLING(SIZE 1 SECOND) GROUP BY ROWKEY;",
+        "SELECT COUNT, ROWTIME + 10 FROM AGGREGATE WHERE ROWKEY='10' AND WindowStart=12000;"
```

Comment:
What's so special about WIndowStart that we need to a test that explicitly has this in the where clause?

Reply:
Because it's a query against a windowed table and you can't do a static query on a windowed table without supplying a window start time at the moment.